io.github.qnapi.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<component type="desktop">
+	<id>io.github.qnapi</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-2.0+</project_license>
+	<name>QNapi</name>
+	<summary>A software for downloading and automatic matching subtitles to movies</summary>
+	<description>
+		<p>A software for downloading and automatic matching subtitles to movies. Works on Linux, OSX and Windows.</p>

```

Comment:
Nitpick: Something similar to http://qnapi.github.io/#description would be better than mentioning it works on OSes Flathub doesn't support.

Reply:
I will try to do something tomorrow based on this.